Gruntfile.coffee

```diff
@@ -90,20 +90,13 @@ This file is generated by `grunt build`, do not edit it by hand.
       options:
         base: 'public',
         message: "Updated to new Chosen version #{version()}"
-      'gh-pages':
-        src: ['**']
-      'bower':
```

Comment:
We opted to use a script instead of gh-pages as it was easier to avoid tagging on every push.